types/bit_vec/module.f.ts

```diff
@@ -1,3 +1,11 @@
+/**
+ * MSb is most-significant bit first.
+ * - byte: 0x53 = 0b0101_0011
+ * -                0123_4567
```

Comment:
Spaces are removed from the documentation, so there will be no alignments. I would suggest to use code blocks:

- https://www.markdownguide.org/extended-syntax/#fenced-code-blocks
- https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#fenced-code-blocks